pkg/api/pvc.go

```diff
@@ -0,0 +1,63 @@
+package api
+
+import (
+	"fmt"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"strings"
+)
+
+var (
```

Comment:
Unsure about this, feels more like you could just return this directly from the `CreatePersistentVolume`? I have never seen it done this way before, do you know if this considered correct?

Reply:
@paulbes I have no idea really. I just wanted to seperate template from the actual values to make it obvious and easy to adjust default values in the future. That way you don't have to touch the implementation when adjusting defaults. 

If you don't agree I can change it. Let me know

Reply:
Right, I get the idea, and I think that makes a lot of sense. What makes me nervous is that you can modify this variable from within the scope of this package, perhaps it is better to provide a `func defaultVolumeTemplate() ... {}` and return the struct from there?

It is just tingling my spidey senses, I am not sure that there is anything fundamentally wrong with this approach